scripts/globals/mobskills/arcane_stomp.lua

```diff
@@ -17,11 +16,6 @@ function onMobSkillCheck(target,mob,skill)
 end;
 
 function onMobWeaponSkill(target, mob, skill)
-
-    local duration = 300;
-    local typeEffect = dsp.effect.MAGIC_SHIELD;
-    local msg = MobBuffMove(mob,typeEffect,3,0,duration);
-
-    skill:setMsg(msg);
-    return typeEffect;
+    skill:setMsg(MobBuffMove(mob,dsp.effect.MAGIC_SHIELD,3,0,300));
+    return dsp.effect.MAGIC_SHIELD;
```

Comment:
consistency and not having variables we never needed is a plus to me. thought normally I don't do this unless I hafta touch a script anyways, he's prolly doing that because of prior examples I showed him.

Reply:
exactly right, while im in there checking over the mob change everything to what it should be like for Teo's OCD

Reply:
My OCD thanks you, even as it prepares to kick me in the behind when I ain't lookin